config/secrets.yml

```diff
@@ -18,11 +18,14 @@
 # Environmental secrets are only available for that specific environment.
 
 development:
+  emotion_api_key: 22e91cf5f1274dd18152bbde46d60dfb
```

Comment:
We don't have to push API Keys to Github repository. Please add `secrets.yml` to `.gitignore` and push `secrets.yml.sample` file with this line replaced to `emotion_api_key: EMOTION_API_KEY`